### PR TITLE
Fix some minor spelling and grammatical errors on the collaborations …

### PR DESCRIPTION
**Pull request by NoahHeinrich**
…index page

### CODE REVIEW

File: app/views/collaborations/index.html.erb [hunk 1]
```diff
@@ -28,11 +28,10 @@ HEREDOC
 <div id="no_collaborations_message" style="<%= hidden unless @collaborations.empty? %>">
   <% if can_do(@context, @current_user, :manage_content) %>
     <p><%= mt :description_teacher, <<-HEREDOC, :google_url => "http://docs.google.com"
-**What do we mean by collaborations?**  We're talking about web-based tools that most
-likely your students are already familiar with.  Students can use resources like
+**What do we mean by collaborations?**  We're talking about web-based tools that your students are likely already familiar with.  Students can use resources like
 [Google Docs](%{google_url})
-to work collaboratively on tasks like group papers or note-taking.  This page gives them
-(and you) an easy place to keep track of those collaborations, and also to set them
+collaborate on tasks like group papers or note-taking.  This page gives them
```
Comment (simonista) [inline app/views/collaborations/index.html.erb]: This still needs to be "to collaborate", right?

File: app/views/collaborations/index.html.erb [hunk 2]
```diff
@@ -85,7 +84,7 @@ HEREDOC
         <div class="collaboration <%= collaboration.style_class %> collaboration_<%= collaboration.id %>" data-id="<%= collaboration.id %>">
           <h3><%= collaboration.title %></h3>
           <div style="margin-bottom: 5px;" class="description">
-            <a href="<%= oauth_url(:service => :google_drive, :return_to => (request.url)) %>"><%= t 'google.drive.upgrade.description', "To access this collaboration you must authorize Canvas to access your Google Drive account" %></a>
+            <a href="<%= oauth_url(:service => :google_drive, :return_to => (request.url)) %>"><%= t 'google.drive.upgrade.description', "To access this Collaboration you must authorize Canvas to access your Google Drive account" %></a>
```
Comment (simonista) [inline app/views/collaborations/index.html.erb]: what's the argument for capitalization here?  seems like this is referring to a generic "collaboration"

Comment (NoahHeinrich) [inline app/views/collaborations/index.html.erb]: Consistency. There are other parts of the page that refer to it with the capitol letter.

File: app/views/collaborations/index.html.erb (hunk 1 above)
Comment (NoahHeinrich) [inline app/views/collaborations/index.html.erb]: You're right, I missed that.

File: app/views/collaborations/index.html.erb (hunk 2 above)
Comment (jenseng) [inline app/views/collaborations/index.html.erb]: -1 ... For a heading or button, it's appropriate to use title case and capitalize nouns/adjectives. However for a paragraph, we should stick with sentence case and only capitalize proper nouns (or the first word).

**Comment by tierra** [comment]
Any plans to continue pushing changes through on this @NoahHeinrich ?

**Comment by simonista** [comment]
Closing due to lack of followup.

**Pull request closed without merging**